src/qnvimcore.cpp

```diff
@@ -727,9 +685,12 @@ void QNVimCore::editorOpened(Core::IEditor *editor) {
     mNumbersColumn->setEditor(textEditor);
 
     widget->setAttribute(Qt::WA_KeyCompression, false);
-    widget->installEventFilter(this);
 
-    QTimer::singleShot(100, this, [=]() { fixSize(editor); });
```

Comment:
Don't we need to call fixSize in the initialization, previously we used to call it on initialization as well as whenever `Resize` event was triggered.